packages/3dom/package.json

```diff
@@ -59,7 +59,7 @@
     "typescript": "3.7.3"
   },
   "dependencies": {
-    "three": "^0.115.0"
+    "three": "cdata/three.js#track-gltf-associations"
```

Comment:
Are we planning to wait for this to be released by three before we merge this PR?

Reply:
I am hoping for the Three.js change to land promptly. We will have to discuss what to do for the next `<model-viewer>` release. If it goes out before the next Three.js release, then we will have to hold on to a forked `GLTFLoader` implementation until we can land the Three.js release update.

Reply:
I think they're currently scheduled for about 1 day apart. I think I'd vote to hold a day or two and pull the release version (assuming this lands in r117).

Reply:
The PR is merged and I have update this change to refer to the Three.js dev branch.